src/base/ParserFaults.messages

```diff
@@ -7,6 +7,9 @@
 #@ WARNING:
 #@ The following comment has been copied from "src/base/ParserFaults.messages".
 #@ It may need to be proofread, updated, moved, or removed.
+#@ WARNING:
+#@ The following comment has been copied from "src/base/ParserFaults.messages".
+#@ It may need to be proofread, updated, moved, or removed.
```

Comment:
Do we need this warning twice?

Reply:
This has been auto-generated by Menhir, I just removed those warnings altogether by hand